src/main/java/com/codeborne/selenide/SelenideElement.java

```diff
@@ -81,6 +81,25 @@ public interface SelenideElement extends WebElement, WrapsDriver, WrapsElement,
   @CanIgnoreReturnValue
   SelenideElement setValue(SetValueOptions options);
 
+  /**
+   * Mimic how real user would type in a text field
+   * Useful to work with autosuggestion dropdown
+   * @see com.codeborne.selenide.commands.Type
+   */
+  @Nonnull
+  @CanIgnoreReturnValue
+  SelenideElement type(CharSequence textToType);
+
+  /**
+   *
```

Comment:
You need at least a link to the first method.

```suggestion
   * Similar to {@link #type(java.lang.CharSequence)}
```